client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigationInfoBar.tsx

```diff
@@ -43,17 +45,17 @@ const defaultInvestigationStaticInfo = {
     userByLastUpdator: defaultUser
 }
 
-const InvestigationInfoBar = (props: Props) => {
+const InvestigationInfoBar = () => {
 
     let history = useHistory();
-    const { epidemiologyNumber } = props;
 
     const [investigationStaticInfo, setInvestigationStaticInfo] = React.useState<InvestigationInfo>(defaultInvestigationStaticInfo);
 
-    React.useEffect(() => {
-        axios.post('/investigationInfo/staticInfo', {
-            investigationId: epidemiologyNumber
-        }).then((result: any) => {
+    const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
+
+    React.useEffect(() => { 
+        axios.get(`/investigationInfo/staticInfo?investigationId=${epidemiologyNumber}`
+        ).then((result: any) => {
             if (result && result.data && result.data.data && result.data.data.investigationByEpidemiologyNumber) {
```

Comment:
the result and result.data can't be undefined I think, because these are the stuff that returns in any axios request

Reply:
not exactly because if the request is unauthorised it returns undefined (i checked it)

Reply:
the result itself is also undefined? but also if you are unauthorized it will go the catch block because it's not a 200 status